src/style.css

```diff
@@ -0,0 +1,103 @@
+@import url(./stars.css);
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 198px;
```

Comment:
Change the max-width of the card to 200px, as specified in the task description.

Reply:
just add box-sizing